src/app/public/modules/url-validation/url-validation.directive.ts

```diff
@@ -20,6 +20,13 @@ const SKY_URL_VALIDATION_VALIDATOR = {
   multi: true
 };
 // tslint:enable
+
+/**
+ * Creates an input to validate URLs. Place this attribute on an `input` element.
+ * If users enters values that are not valid URLs, an error message appears.
+ * This directive uses `NgModel` to bind data.
+ * @Required
```

Comment:
Does the `@Required` do anything in this situation? I'm pretty sure `@required` is only recognized for properties and parameters...